src/db/writeMessage.js

```diff
@@ -0,0 +1,33 @@
+const uuid = require('uuid/v4')
```

Comment:
```
(node:99) DeprecationWarning: Deep requiring like `const uuidv4 = require('uuid/v4');` is deprecated as of uuid@7.x. Please require the top-level module when using the Node.js CommonJS module or use ECMAScript Modules when bundling for the browser. See https://github.com/uuidjs/uuid#deep-requires-now-deprecated for more information.
```

Who knew

Reply:
You know what, I saw that warning earlier this week in a node REPL, and it never occurred to me to do something about it in the actual code.  I'll fix that up.